code/__DEFINES/cargo.dm

```diff
@@ -47,3 +47,17 @@
 #define EXPORT_SOLD 1
 ///Sell the item, but for the love of god, don't delete it, we're handling it in a fancier way.
 #define EXPORT_SOLD_DONT_DELETE 2
+
+
+//At 320 it's 9.8 minutes, at 1400 it's 13.18 minutes,  at 3000 (around gun crates) it's 14.5 minutes, at 8000 (hat crate) 17 minutes, at 9000 (expensive atmos cans) it's 17.3 minutes, and at the 20k crate it's 19.18 minutes.
+/// Multiplies the logarithmic value calculating the free crate cooldown
+#define DEPARTMENTAL_ORDER_COOLDOWN_COEFFICIENT 200
+/// Used for the power of the logarithmic value for the free crate cooldown
+#define DEPARTMENTAL_ORDER_COOLDOWN_POWER 1.2
```

Comment:
Based on discussion on discord, proposing changing the coefficient to 60 and exponent to 2.2, so as to not punish players for requesting less expensive crates.

Reply:
I shall do so.

Reply:
Also I completely forget that is what it is called in english, changing power to exponent in the define names.